js/Header.js

```diff
@@ -4,7 +4,15 @@ import { Link } from "react-router"
 export default React.createClass({
   render() {
     return(
-      <h1>This is a header</h1>
+      <section className="headerBar">
```

Comment:
I think it would make more sense to use a `<header>` tag here to make it explicit.